pkg/errors/errors.go

```diff
@@ -26,6 +26,9 @@ var (
 
 	// ErrInternalServerError is raised when an internal server error or similar is received
 	ErrInternalServerError = fmt.Errorf("internal server error, please try again")
+
+	// ErrQuota is returned when there aren't enough resources to enable dev mode
+	ErrQuota = fmt.Errorf("Exceeded quota, please adjust your limits and resources and try again")
```

Comment:
or release a few resources too?

Reply:
@rlamana what do you think it's a good error message here?